src/ZenstruckFoundryBundle.php

```diff
@@ -114,6 +114,16 @@ public function configure(DefinitionConfigurator $definition): void
                                     ->defaultValue(AbstractORMPersistenceStrategy::RESET_MODE_SCHEMA)
                                     ->values([AbstractORMPersistenceStrategy::RESET_MODE_SCHEMA, AbstractORMPersistenceStrategy::RESET_MODE_MIGRATE])
                                 ->end()
+                                ->arrayNode(AbstractORMPersistenceStrategy::RESET_MODE_MIGRATE)
+                                    ->addDefaultsIfNotSet()
+                                    ->children()
+                                        ->arrayNode('configurations')
+                                            ->info('Migration configurations')
+                                            ->defaultValue([])
+                                            ->scalarPrototype()->end()
```

Comment:
Could you add a check on the existence of the files?

Reply:
Done

Reply:
yes, I'm wondering if you should not concat with `kernel.project_dir` or something like this. But I'm wondering if this param is accessible here

Reply:
Seems like parameters value have to be injected: https://symfony.com/doc/current/configuration/using_parameters_in_dic.html

Not sure how to do it in a bundle class though.

Reply:
I think you can just use `$container->getParameter('kernel.project_dir')`

Reply:
maybe we can change the strategy and require absolute paths?
and we'd accept `'%kernel.project_dir%/path/to/conf.php` as well has `/aboslute/path/to/conf.php`

Reply:
I was looking at e.g. Symfony translator’s `default_path` and there is no restriction on whether the path is absolute or relative, but as there is no reliable way to resolve a relative path the recipe simply provides an absolute path:

```yaml
framework:
    translator:
        default_path: '%kernel.project_dir%/translations'
```

That would be the simplest and perhaps more intuitive way :thinking:

Reply:
yeah I think we agree on this, let's not enforce anything, and just use `file_exist($migrationConfiguration)` at runtime.

We'll add an example using `%kernel.project_dir%` in the docs

Reply:
The check can still be done at compile time since parameters will be resolved when the `configurations` node validates.

Should I move it into `PersistenceStrategy`’s constructor?

Reply:
I'd say the sooner the better! if the params are resolved in the validation phase, let's keep it here